exercises/python-2-a/python_2_a_test.py

```diff
@@ -1 +1,96 @@
-# TODO
+"""
+Tests for python-2-a exercise.
+"""
+import unittest
+import string
+from itertools import chain
+
+from python_2_a import compress, decompress
+
+
+class TestCompress(unittest.TestCase):
+    def assertCanCompressAndDecompress(self, string, expect, slug):
```

Comment:
This is the only thing I'm not sure of in this PR. It means that all existing tests are now no longer valid. I'm not sure if that is an issue, but I'd like to check with @iHiD.

Reply:
Well, to be fair, the previous tests were doing the exact same thing, they just had a `_decompress` implementation built into them which was verifying that the student's compression could roundtrip ... now it's verifying the exact same thing, only they've got to provide the working implementation.

Reply:
Right! Then it's perfectly okay I think. I'll merge this and add the other documents later.